release_checklist/patch/index.md

```diff
@@ -10,49 +10,75 @@ These are the steps involved in performing a Mantid patch release.  To perform a
 
-# Release
+* find the list of commit `SHA1` values in that pull request
+* check if any of these commits has altered the release notes for the next major release
+  * if not then pass all commits in the order oldest->newest to `git cherry-pick -x`
+  * if release notes were modified in a commit on their own then pass all commits except this one in the order oldest->newest to `git cherry-pick -x`
+  * if a commit has a mixture of code/release note changes then:
+    * pass the list of commits up to but not including this commit to `git cherry-pick -x`
+    * now pass this single commit to `git cherry-pick -x -n` and it will not make a commit. Remove the major release note changes and commit `git add`/`git commit`
+    * if there are any remaining commits after this then pass them to `git cherry-pick -x` as before.
+* finally add a commit that updates the patch release notes with this pull request link and description in the table.
 
-Once the unscripted testing has passed and the Release Notes are complete:
-* Create a new release on GitHub, which will create a new version tag
-* Turn on the release build jobs on jenkins and update the branches for all of the clean builds to point to the correct branch.
-* Turn on the release system test jobs and update the branch to point to the tag of the last major release.
-* Wait for builds to complete
-* Put packages on GitHub & sourceforge
+Once cherry picked the milestone of the original pull request should be updated to the patch milestone.
+
+# Nightly Builds
+
+The [Jenkins Release Pipeline]({{ site.mantidreleasebuilds }}) contains jobs that check for changes on the current release branch each night (00:00 GMT). Any detected changes will cause a clean build of the code followed by a run of the system tests. These jobs should be checked each morning to confirm that everything is green.
+
+# Release Day
+
+On the day of release a few steps are required:
+
+* update the patch version:
+  * navigate to {{site.mantidrepo}}/blob/release-X.Y./buildconfig/CMake/VersionNumber.cmake, where `X` & `Y` are the major and minor release versions respectively.
+  * edit the `VERSION_PATCH` to the required number for the patch and commit the result.
+* run a manual build of all of the OS jobs under {{ site.mantidreleasebuilds }} and when asked for a suffix use an empty string
+* wait for the builds to finish (will take more than 1 cup of tea/coffee/beverage of choice)
+
+While waiting for the builds create a new release on GitHub, using a tag of the form `v.X.Y.Z` and populate with information from the release notes (see a previous version of the format).
+
+Once the builds complete have the development team run unscripted testing on the packages generated by the clean release builds. In particular the issues intended to be fixed should be tested.
+
+Once the testing has passed:
+
+* Use the manual deploy job at {{ site.mantidreleasebuilds }} to deploy packages and documentation to the public web.
```

Comment:
The `SUFFIX` package (currently for only RHEL7) needs to be mentioned either here or in the regular release checklist